docs/changelog.json

```diff
@@ -1,28 +1,204 @@
 {
   "url": "https://github.com/Shopify/hydrogen/pulls?q=is%3Apr+is%3Aclosed+%5Bci%5D+release+in%3Atitle+is%3Amerged",
-  "version": "2024.4.0",
+  "version": "2024.4.2",
   "releases": [
+    {
+      "title": "Dependencies update to avoid React version mismatches",
+      "version": "2024.4.2",
+      "hash": "0131d8d0abe471fec458a1211a1ba8eda01f2403",
+      "pr": "https://github.com/Shopify/hydrogen/pull/2060",
+      "commit": "https://github.com/Shopify/hydrogen/pull/2060/commits/0131d8d0abe471fec458a1211a1ba8eda01f2403",
+      "dependencies": {
+        "@remix-run/react": "^2.8.0",
+        "@remix-run/server-runtime": "^2.8.0",
+        "@shopify/cli": "3.59.2",
+        "@shopify/cli-hydrogen": "^8.0.3",
+        "@shopify/hydrogen": "2024.4.1",
+        "@shopify/remix-oxygen": "^2.0.4",
+        "graphql": "^16.6.0",
```

Comment:
just so I know, whats the situation with TS in which it was added into the devDependencies going forward?

Reply:
If I don't remember wrong, it's because TS can make breaking changes in minor releases. And perhaps we started using a new feature released in Hydrogen so we had to bump the minimum version in user projects 🤔